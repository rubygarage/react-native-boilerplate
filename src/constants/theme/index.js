export default {
  colors: {
    black: '#000000',
    white: '#ffffff',
  },
  fonts: {
  },
  sizes: {
  },
};
