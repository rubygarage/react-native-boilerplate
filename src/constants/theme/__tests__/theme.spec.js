import theme from '../index';

describe('theme', () => {
  it('matches snapshot', () => {
    expect(theme).toMatchSnapshot();
  });
});
