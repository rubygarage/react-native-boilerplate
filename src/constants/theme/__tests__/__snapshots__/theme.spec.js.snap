// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme matches snapshot 1`] = `
Object {
  "colors": Object {
    "black": "#000000",
    "white": "#ffffff",
  },
  "fonts": Object {},
  "sizes": Object {},
}
`;
